src/main/java/com/github/lwhite1/tablesaw/api/Table.java

```diff
@@ -35,651 +38,679 @@
+            e.printStackTrace();
+        }
+    }
+
+    public String save(String folder) {
+        String storageFolder = "";
+        try {
+            storageFolder = StorageManager.saveTable(folder, this);
+        } catch (IOException e) {
+            System.err.println("Unable to save table in Tablesaw format");
+            e.printStackTrace();
+        }
+        return storageFolder;
+    }
+
+    public static Table readTable(String tableNameAndPath) {
+        Table t;
+        try {
+            t = StorageManager.readTable(tableNameAndPath);
+        } catch (IOException e) {
+            System.err.println("Unable to load table from Tablesaw table format");
+            e.printStackTrace();
+            return null;
+        }
+        return t;
+    }
+
+    /**
+     * Returns the result of applying the given function to the specified column
+     *
+     * @param numericColumnName The name of a numeric (integer, float, etc.) column in this table
+     * @param function          A numeric reduce function
+     * @return the function result
+     * @throws IllegalArgumentException if numericColumnName doesn't name a numeric column in this table
+     */
+    public double reduce(String numericColumnName, NumericReduceFunction function) {
+        Column column = column(numericColumnName);
+
+        return function.reduce(column.toDoubleArray());
+    }
+
+    //TODO(lwhite): Reorder params so the grouping param can be an array
+    public Table reduce(String numericColumnName, String groupColumnName, NumericReduceFunction function) {
+        TableGroup tableGroup = new TableGroup(this, groupColumnName);
+        return tableGroup.reduce(numericColumnName, function);
+    }
+
+    /**
+     * Create a Table from the provided CSV.
+     *
+     * @param columnTypes the data types from the first row of the CSV
+     * @param input the path to the csv to read.
+     * @return a Table instance from the CSV data
+     * @throws IllegalStateException if the CSV data can't be read for some reason.
+     */
+    public static Table fromCSV(ColumnType[] columnTypes, String input) {
+        Table table;
+        try {
+            table = CsvReader.read(columnTypes, input);
+        } catch (IOException e) {
+            throw new IllegalStateException("IO error creating tablesaw from: " + input, e);
+        }
+        return table;
+    }
+
+    /**
+     * Returns the entire row in a table as a CSV string
+     *
+     * @param index the specified row in the table
+     * @return a string which each column value is separated by a comma.
+     * @throws IndexOutOfBoundsException if the index is invalid
+     */
+    public String getRow(int index) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        for (Column column : columnList) {
+            stringBuilder.append(column.getString(index));
+            stringBuilder.append(",");
+        }
+
```

Comment:
you really can't build a csv String without a CSV library (too many issues with escaping quotes, etc.). There is code that does this in the CsvWriter class.  
IMO, this is a utility that should be outside the table class. Maybe in a test utilities class?